### ISSUE TITLE
Query different timestamp formats

### ISSUE THREAD

**Issue by RinkAttendant6** [node-influx/node-influx#111]
I can't find any way of doing this with node-influx. Is it possible to implement it in the `query` and `queryRaw` methods?

> #### Timestamp Format
> Everything in InfluxDB is stored and reported in UTC. By default, timestamps are returned in RFC3339 UTC and have nanosecond precision, for example `2015-08-04T19:05:14.318570484Z`. If you want timestamps in Unix epoch format include in your request the query string parameter `epoch` where `epoch=[h,m,s,ms,u,ns]`. For example, get epoch in seconds with:
> ```shell
curl -G 'http://localhost:8086/query' --data-urlencode "db=mydb" --data-urlencode "epoch=s" --data-urlencode "q=SELECT value FROM cpu_load_short WHERE region='us-west'"
```

https://influxdb.com/docs/v0.9/guides/querying_data.html#timestamp-format

**Comment by bencevans** [comment]
Thanks for reporting @RinkAttendant6. You're correct, this module currently doesn't support passing additional options as part of query requests. Please see https://github.com/node-influx/node-influx/issues/112 as  there's another issue open related, we can aim to fix both.

**Comment by connor4312** [comment]
Resolved in 5.x

**Comment by connor4312** [comment]
These changes have been published in `5.0.0-alpha.1`. You can install this via:

```
npm install --save influx@next
```
